app/CocoaPods/CPDeintegrateController.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+
+public enum CPDeintegrateErrors: ErrorType {
+  case CommandError(String)
+
+  var message: String {
+    switch self {
+    case .CommandError(let s): return s
+    }
+  }
+}
+
+public class CPDeintegrateController: NSObject, CPCLITaskDelegate {
+  let xcodeprojURL: NSURL
+  private let completionHandler: (CPDeintegrateErrors?) -> ()
+  private var task: CPCLITask!
+  private let output = NSMutableAttributedString()
+
+  init(xcodeprojURL: NSURL, completionHandler: (error: CPDeintegrateErrors?) -> ()) {
```

Comment:
I've not been bothering with `public` at all so far, don't have a strong opinion either way 👍